### PR TITLE
Results by membership type and weight

### PR DESCRIPTION
**Pull request by sauloperez**
## What? Why?

This closes https://github.com/coopdevs/decidim-coopcat/issues/2. 

This modifies the consultation's result page to display the metadata included in users' authorizations. This relies on the following particular format. Things will be slightly off if the metadata doesn't stick to it (there's no enforce schema).

```json
"{ metadata_type: '',   metadata_weight: '' }"
```

## :camera_flash: 

Screenshot from 2020-09-29 14-58-09

### CODE REVIEW

File: app/views/decidim/consultations/admin/consultations/results.html.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              <% end %>
+            <% else %>
+              <tr><td><em><%= t "decidim.admin.consultations.results.not_visible" %></em></td><td>&nbsp;</td></tr>
+            <% end %>
+          </tbody>
+        <% end %>
+      <% end %>
+    </table>
+  </div>
+</div>
```
Comment (sauloperez) [inline app/views/decidim/consultations/admin/consultations/results.html.erb:46]: :warning: I missed to add the override in specs.

File: app/queries/decidim/action_delegator/responses_by_membership.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    class ResponsesByMembership < Rectify::Query
```
Comment (sauloperez) [inline app/queries/decidim/action_delegator/responses_by_membership.rb:5]: better doc how the authorization record should look like for this to work.

**Review by microstudi — APPROVED** [review]
We should add in the README or even in the admin somewhere that this module requires a verification system that has the 2 required metadata names ("membership_type", "membership_weight").
Idea: for the future we might want to set a configuration option for this metadata names.
Or, improve the direct verification module integration with this one

**Comment by sauloperez** [comment]
good point. I'll document in the README something similar to what I did in the query object. I think we'll need to explain this in the text accompanying the text area in admin but that's something that will have to done in here and not in direct_verifications because it's specific to our case.

**Pull request merged**